cmd/todo/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/todannigo/internal/databases/sql"
+	todo2 "github.com/todannigo/internal/repositories/todo"
+	"github.com/todannigo/internal/services/todo"
```

Comment:
The import path here is going to cause some issues (and is what is calling your builds to fail). Ideally you would update these imports to match the gopath:

```suggestion
	"github.com/danni-popova/todannigo/internal/databases/sql"
	todo2 "github.com/danni-popova/todannigo/internal/repositories/todo"
	"github.com/danni-popova/todannigo/internal/services/todo"
```

Also it would be good practice to define a `go.mod` file at the root of your repo, that's the standard for Golang projects since version 1.14. To do this you'd run `go mod init` then follow that up by attempting to build your two applications (`go build cmd/account/main.go` and `go build cmd/todo/main.go`). Once paths are sorted that should add all of your dependencies to the module file.

I'd also suggest a more semantic alias for the repo rather than `todo2` perhaps `todoRepo`?